umapi_client/functional.py

```diff
@@ -111,31 +82,13 @@ def __init__(self, id_type=IdentityTypes.adobeID, email=None, username=None, dom
             raise ArgumentError("Identity type (%s) must be one of %s" % (id_type, [i.name for i in IdentityTypes]))
         self.id_type = id_type
         self.email = None
-        self.domain = None
-        if username:
-            if email and username.lower() == email.lower():
```

Comment:
yes this check needs to come out, because email and username *can* be specified separately.

Reply:
This might have been sleep deprivation.  I'll take a look and see what needs to happen here.